modules/runtime/nuxeo-stream/src/test/java/org/nuxeo/lib/stream/tests/log/TestLogChronicle.java

```diff
@@ -56,7 +56,9 @@
 
     @Before
     public void skipWindowsThatDoNotCleanTempFolder() {
-        org.junit.Assume.assumeFalse(IS_WIN);
+        // NXP-29161 - https://stackoverflow.com/questions/58733295/why-does-chronicle-queue-hold-on-to-files-for-a-second-after-close
+        // beware: not all tests actually work on Windows
+        // org.junit.Assume.assumeFalse(IS_WIN);
```

Comment:
If you remove everything then you can remove the whole method

Reply:
👍